recipe/bld.bat

```diff
@@ -1,6 +1,5 @@
-set SITECFG=cf_units/etc/site.cfg
-echo [System] > %SITECFG%
-echo udunits2_xml_path = %LIBRARY_PREFIX%\share\udunits\udunits2.xml >> %SITECFG%
+set UDUNITS2_INCDIR="%LIBRARY_PREFIX%\include\udunits2"
+set UDUNITS2_XML_PATH="%LIBRARY_PREFIX%\share\udunits\udunits2.xml"
```

Comment:
@ocefpaf `windows` isn't happy with this... not quite sure what to do about it.

Any ideas?

BTW I changed `PREFIX` to `LIBRARY_PREFIX`, but that's now working :thinking:

Reply:
Thanks. That is correct. I'll take a look on Windows as soon as possible. I'm sending bad PRs left and right lately due to lack of proper time to focus on a single task :-(

Reply:
@ocefpaf No pressure from this side my friend. I'll try to dig and see where I get to.

If you need me, just ping me :+1: